scipy/special/_generate_pyx.py

```diff
@@ -135,14 +135,30 @@
 """
 
 STUBS = """\
-from typing import Any, Type
+from typing import Any, Dict
 
 import numpy as np
 
 __all__ = [
+    'geterr',
+    'seterr',
+    'errstate',
     {ALL}
 ]
 
+def geterr() -> Dict[str, str]: ...
```

Comment:
Originally I tried something much fancier here with `TypedDict` and `Literal`, but then our own tests didn't satisfy something that strict, so I went with the looser version.